config/routes.rb

```diff
@@ -22,9 +22,13 @@ def redirect(path, options = { prefix: Whitehall.router_prefix })
 
       resources :users, only: %i[index show edit update]
 
-      get "republishing" => "republishing#index", as: :republishing_index
-      get "republishing/:document_slug/confirm" => "republishing#confirm_commonly_republished_document", as: :confirm_commonly_republished_document_republishing
-      post "republishing/:document_slug/republish" => "republishing#republish_commonly_republished_document", as: :republish_commonly_republished_document
+      scope :republishing do
+        root to: "republishing#index", as: :republishing_index, via: :get
+        scope :document do
```

Comment:
This routing makes sense, but I wonder if it conflicts with our understanding of this entity as a "document" vs it being "content"? I think we need to decide what to call these different types of "thing" being published as it'll help with the other question I had re: what to call the things we're republishing.